lib/tasks/petitions.rake

```diff
@@ -32,7 +32,7 @@ namespace :wpets do
 
     desc "Backfill moderation lag"
     task :backfill_moderation_lag => :environment do
-      %w[petitions archived_petitions].each do |table_name|
+      %w[petitions].each do |table_name|
```

Comment:
no more need for `each`

Reply:
Was being lazy - removing the each means rewriting the code block to remove the interpolation of `table_name` 😄

Reply:
Removed the `each` but kept the dynamic class definition because I don't want to load the `Petition` class - a bit like how we redefine models in migrations for safety, but we don't have a namespace to protect us in a rake task.

Reply:
just thought of `"petitions".tap do |table_name|` but don't listen to me good job :hear_no_evil: